File: POO/src/pooG/javascript/Car.js
function Car(license, drive) {
    this.id;
    this.license = license;
    this.driver = drive;
    this.passenger;
}

Car.prototype.printDataCar = function() {
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}

File: POO/src/pooG/java/Car.java
public class Car {

    private Integer id;
    private String license;
    private Account driver;
    protected Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if(passenger != null)
            System.out.println("License: " + license + "\nDriver: " + driver.name);
    }

    public Integer getPassenger() {
        return passenger;
    }

    public void setPassenger(Integer passenger) {
        if(passenger == 4)
            this.passenger = passenger;
        else
            System.out.println("No se puede establecer ese valor");
    }
}
